tests/platformalteration/helper/helper.go

```diff
@@ -32,3 +38,51 @@ func WaitForSpecificNodeCondition(clients *client.ClientSet, timeout, interval t
 		return false, nil
 	})
 }
+
+// UpdateAndVerifyHugePagesConfig updates the hugepages file with a given number,
+// and verify that it was updated successfully.
+func UpdateAndVerifyHugePagesConfig(updatedHugePagesNumber int, filePath string, pod *corev1.Pod) (bool, error) {
+	cmd := fmt.Sprintf("echo %d > %s", updatedHugePagesNumber, filePath)
+	_, err := globalhelper.ExecCommand(*pod, []string{"/bin/bash", "-c", cmd})
+
```

Comment:
Just my recommendation for this kind of `err := functionCall()` and the following `if err != nil` checks:
```
	cmd := fmt.Sprintf("echo %d > %s", updatedHugePagesNumber, filePath)

	_, err := globalhelper.ExecCommand(*pod, []string{"/bin/bash", "-c", cmd})
	if err != nil {
		return false, err
	}

    timeout := time.Now().Add(5 * time.Minute)
```
It feels more natural (at least in go) to check the error in the following line right after the function call.
The empty line between the `cmd` variable assignment and the function call makes this block easier to read. That will also avoid the linting error you'll get otherwise...

Reply:
thanks, agree, usually i try to stick to this way, missed it.